app/controllers/episodes_controller.rb

```diff
@@ -1,11 +1,48 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
+  before_action :set_episode, only: %i[edit show]
 
   def index
     @episodes = Episode.order(created_at: :desc).page params[:page]
   end
 
-  def show
+  def show; end
+
+  def edit; end
+
+  def create
+    @episode = Episode.new(episode_params.merge(created_by: current_user))
+
+    if @episode.save
+      redirect_to episodes_url, notice: 'Successfully created'
+    else
+      render :new, notice: 'Something wrong'
+    end
+  end
+
+  def update
+    if Episode.find(params[:id]).update(episode_params)
```

Comment:
You use If everywhere. Mb better to raise errors if something goes wrong and rescue from it with so flash? Read about `rescue_from`. This can be added as an improvement later